### DISCUSSION TITLE
为啥挂载的阿里云盘最近总是显示不出来勒？

### DISCUSSION THREAD

**Discussion by zhushen12580** [AlistGo/alist#4476] [Q&A]
如题

**Comment by mustundead** [comment]
我最近几乎每天都要换一次 Token，现在是不让用了么

**Reply by anwen-anyi** [reply]
> 我最近几乎每天都要换一次 Token，现在是不让用了么

你确定你是最新版的？不截图直接说是最新版的都是耍流氓
你确定你是最新版的？不截图直接说是最新版的都是耍流氓

**Reply by lovekidd** [reply]
我的也是，更新了最新版也是，每天都要换一次Token，今天直接说Too many requests，不知道怎么回事

**Reply by zhoutao-cn** [reply]
你们是不是在 KODI 里打开了刮削？
